File: HeartForCharity/HeartForCharity.Model/Requests/CampaignMediaUpsertRequest.cs
namespace HeartForCharity.Model.Requests
{
    public class CampaignMediaUpsertRequest
    {
        public int CampaignId { get; set; }
        public string Url { get; set; } = null!;
        public string MediaType { get; set; } = "Image";
        public bool IsCover { get; set; } = false;
    }
}

File: HeartForCharity/UI/heartforcharity_desktop/lib/model/requests/campaign_media_upsert_request.dart
class CampaignMediaUpsertRequest {
  final int campaignId;
  final String url;
  final String mediaType;
  final bool isCover;

  CampaignMediaUpsertRequest({
    required this.campaignId,
    required this.url,
    this.mediaType = 'Image',
    this.isCover = false,
  });

  Map<String, dynamic> toJson() => {
        'campaignId': campaignId,
        'url': url,
        'mediaType': mediaType,
        'isCover': isCover,
      };
}
